tasks.py

```diff
@@ -88,6 +88,12 @@ def docker_compose(context, command, **kwargs):
         compose_file_path = os.path.join(context.nautobot_chatops.compose_dir, compose_file)
         compose_command += f' -f "{compose_file_path}"'
     compose_command += f" {command}"
+
+    # If `service` was passed as a kwarg, add it to the end.
+    service = kwargs.pop("service", None)
+    if service is not None:
+        compose_command += f" {service}"
+
```

Comment:
Can I have more details on why this is needed for the invoke tasks?

Reply:
This was not a change I made.  I've no idea where that came from unless it was a relic of the rebase

Reply:
Oh, that.  This was added so that I could do an import of the gizmo database into my demo environment.  It allows us to do `invoke start --service=postgres`.

Copy the data to the nautobot_postgres container
```
(nautobot-6LHZxDho-py3.8) PycharmProjects/nautobot % docker cp ../gizmo-nautobot/nautobot_backup.dump nautobot_postgres_1:/tmp/                                                                                   
(nautobot-6LHZxDho-py3.8) PycharmProjects/nautobot %  
```

Drop to the nautobot_postgres container bash shell

```
(nautobot-6LHZxDho-py3.8) PycharmProjects/nautobot % docker exec -it nautobot_postgres_1 bash                                                                                                                     
root@4dd571152b04:/# 


root@4dd571152b04:/# psql -U nautobot -W < /tmp/nautobot_backup.dump 
Password: <# POSTGRES_PASSWORD from nautobot/development/dev.env >
SET
SET
SET
<snip>
```

I think this is a huge value add because it lets us import data into our chatops demo environment.